nix/status-go/build-status-go.nix

```diff
@@ -53,11 +53,8 @@ let
     '';
 
     # replace hardcoded paths to go package in /nix/store, otherwise Nix will fail the build
-    preFixup = ''
-      ${preFixup}
-
+    preFixup = preFixup + ''
       find $out -type f -exec ${removeExpr removeReferences} '{}' + || true
-      return
```

Comment:
Why did you remove this `return` here? It is required, otherwise the host `preFixup` script will continue executing, which we don't want. Maybe we can add a comment to make it explicit.